frontned/src/components/navbar/DropdownMenu.tsx

```diff
@@ -0,0 +1,121 @@
+import { useState } from 'react';
+import { MouseEvent } from 'react';
+import { styled } from '@mui/material/styles';
+import Button from '@mui/material/Button';
+import Menu, { MenuProps } from '@mui/material/Menu';
+import MenuItem from '@mui/material/MenuItem';
+import KeyboardArrowDownIcon from '@mui/icons-material/KeyboardArrowDown';
+import { Divider } from '@mui/material';
+
+const StyledMenu = styled((props: MenuProps) => (
```

Comment:
Move all styled components to the bottom of the page. 
We set them once and don't wont to distract from main component template ;)